src/components/Pagination.stories.tsx

```diff
@@ -18,34 +14,29 @@ export default {
       url: "https://www.figma.com/file/aWUE4pPeUTgrYZ4vaTYZQU/%E2%9C%A8Beam-Design-System?node-id=3214%3A10519",
     },
   },
-  decorators: [withBeamDecorator],
 } as Meta;
 
 export function WithPages() {
-  return <RenderPagination totalRows={999} label="items" />;
```

Comment:
I removed label b/c it seems noisy and needlessly clutters the UI

Reply:
Yeah, good idea, I'll put it on the list to go find the canonical pagination figma and make a comment...

Reply:
(https://www.figma.com/file/aWUE4pPeUTgrYZ4vaTYZQU/%E2%9C%A8Beam-Component-Library?type=design&node-id=32547-99453&t=pskEtucLTxhgvJE4-0)

Reply:
Ah thanks! Actually looking at those mocks, I don't think they have a per-page `label` anyway, like it just says "items" which imo sounds pretty generic. So I think this actually makes the component more like the figma than less. :tada: :-)